sdcm/cluster.py

```diff
@@ -4989,7 +4989,7 @@ def install_scylla_monitoring_prereqs(node):  # pylint: disable=invalid-name
                 sh get-docker.sh
                 systemctl start docker
             """)
-        elif node.is_ubuntu():
+        elif node.distro.is_ubuntu18 or node.distro.is_ubuntu16:
```

Comment:
ubuntu 16 is deprecated and not supported anymore (only on old branches, and i expect this commit will never get to these branches, please remove it

Reply:
@rayakurl , it is still supported to the "old" released versions of Scylla, but we dropped the support on newer versions